docs/features/models/SQuAD.ipynb

```diff
@@ -0,0 +1,679 @@
+    "\n",
+    "    3.1. [Predict using Python](#3.1-Predict-using-Python)\n",
+    "    \n",
+    "    3.2. [Predict using CLI](#3.2-Predict-using-CLI)\n",
+    "     \n",
+    "4. [Train the model on your data](#4.-Train-the-model-on-your-data)\n",
+    "    \n",
+    "    4.1. [from Python](#4.1-Train-your-model-from-Python)\n",
+    "    \n",
+    "    4.2. [from CLI](#4.2-Train-your-model-from-CLI)\n",
+    "    \n",
+    "5. [Evaluate](#5.-Evaluate)\n",
+    "    \n",
+    "    5.1. [from Python](#5.1-Evaluate-from-Python)\n",
+    "    \n",
+    "    5.2. [from CLI](#5.2-Evaluate-from-CLI)\n",
+    "    \n",
+    "6. [Models list](#6.-Models-list)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "6rlEvcLSpBpe"
+   },
+   "source": [
+    "# 1. Introduction to the task"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "t1HDPkc-pBpe"
+   },
+   "source": [
+    "Context Question Answering is a task of finding a fragment with an answer to a question in a given segment of context."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Context**:"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "YbU1PKjD8Ho3"
+   },
+   "source": [
+    "```\n",
+    "In meteorology, precipitation is any product of the condensation \n",
+    "of atmospheric water vapor that falls under gravity. The main forms \n",
+    "of precipitation include drizzle, rain, sleet, snow, graupel and hail… \n",
+    "Precipitation forms as smaller droplets coalesce via collision with \n",
+    "other rain drops or ice crystals within a cloud. Short, intense periods \n",
+    "of rain in scattered locations are called “showers”.\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "NtJfjLT3pBpf"
+   },
+   "source": [
+    "**Question**:\n",
+    "```\n",
+    "Where do water droplets collide with ice crystals to form precipitation?\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "XH6JS7bs8Ho5"
+   },
+   "source": [
+    "**Answer**: \n",
+    "```\n",
+    "within a cloud\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "vEjlhWiNpBpi"
+   },
+   "source": [
+    "Datasets that follow this task format:\n",
+    "\n",
+    "- [Stanford Question Answering Dataset (SQuAD) (EN)](https://rajpurkar.github.io/SQuAD-explorer/)\n",
+    "\n",
+    "- [SberQuAD (RU)](https://paperswithcode.com/dataset/sberquad)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "NzHeUOeqpBpk"
+   },
+   "source": [
+    "# 2. Get started with the model"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "ygOmNC9R9l_y"
+   },
+   "source": [
+    "First make sure you have the DeepPavlov Library installed.\n",
+    "[More info about the first installation](http://docs.deeppavlov.ai/en/master/intro/installation.html)"
```

Comment:
"[More info about the first installation.](http://docs.deeppavlov.ai/en/master/intro/installation.html)"